tldw_Server_API/tests/Embeddings/test_vector_compactor.py

```
@@ -82,3 +83,58 @@ async def _fake_ids(dbp):
+        finally:
+            db.close_connection()
+
+    monkeypatch.setattr(vc, "managed_media_database", _fake_managed_media_database, raising=False)
+    monkeypatch.setattr(
+        vc,
+        "create_media_database",
+        lambda **kwargs: (_ for _ in ()).throw(AssertionError("legacy raw factory should not be used")),
+        raising=False,
+    )
+
+    result = asyncio.run(vc._get_media_ids_marked_deleted("/tmp/vector-compactor.db"))
```

Ruff (0.15.6)

[error] Probable insecure usage of temporary file or directory: "/tmp/vector-compactor.db"

(S108)